libr/util/format.c

```diff
@@ -506,7 +506,8 @@ static void r_print_format_time(const RPrint* p, int endian, int mode,
 	if (MUSTSET) {
 		p->cb_printf ("wv4 %s @ 0x%08"PFMT64x"\n", setval, seeki+((elem>=0)?elem*4:0));
 	} else if (MUSTSEE) {
-		char *timestr = strdup(asctime (gmtime_r ((time_t*)&addr, &timestruct)));
+		char *timestr = malloc(26 * sizeof(char));
```

Comment:
missing space before (. and sizeof(char) is always1, so there's no need to add such multiplication